src/main/java/gov/usds/case_issues/config/WebConfig.java

```diff
@@ -44,14 +44,21 @@
 	@Override
 	public void addCorsMappings(CorsRegistry registry) {
 		String[] origins = _customProperties.getCorsOrigins();
+		// NOTE: Spring Data Rest (/resources) CORS configuration is in RestConfig, not here.
 		LOG.info("Configuring CORS allowed origins for API to {}", Arrays.toString(origins));
 		if (origins != null && 0 < origins.length) {
-			registry
-				.addMapping("/api/**")
+			registry.addMapping("/api/**")
 					.allowCredentials(true)
 					.allowedMethods("*")
-					.allowedOrigins(origins)
-			;
+					.allowedOrigins(origins);
+			registry.addMapping("/user")
+					.allowCredentials(true)
+					.allowedMethods("GET")
+					.allowedOrigins(origins);
```

Comment:
This is going to allow API access to `/user`? ls the long-term plan to create a new endpoint within `api` that would just provide the information the front-end needs?

Reply:
Yeah, I’m not sure what we’ll end up with but this probably shouldn’t have been made API-accessible in any case.